apps/guardian-ui/src/languages/en.json

```diff
@@ -1,8 +1,10 @@
 {
   "common": {
+    "bitcoin": "Bitcoin",
```

Comment:
nit: when used as a measurement or unit, it would be `bitcoin`. `Bitcoin` with a capital B is for the network or protocol. (I'll take "things that don't matter that much to anyone else" for 500, Alex)